docs/core/testing/mstest-analyzers/mstest0007.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "MSTEST0007: Ensure test attributes can only be set on methods marked with TestMethod attribute"
+description: "Learn about code analysis rule MSTEST0007: Ensure test attributes can only be set on methods marked with TestMethod attribute"
+ms.date: 02/01/2024
+f1_keywords:
+- MSTEST0007
+- UseAttributeOnTestMethodAnalyzer
+helpviewer_keywords:
+- UseAttributeOnTestMethodAnalyzer
+- MSTEST0007
+author: cvpoienaru
+ms.author: codrinpoienaru
+dev_langs:
+- CSharp
+- VB
+---
+# MSTEST0007: Ensure test attributes can only be set on methods marked with TestMethod attribute
+
+| Property                            | Value                                                                              |
+|-------------------------------------|------------------------------------------------------------------------------------|
+| **Rule ID**                         | MSTEST0007                                                                         |
+| **Title**                           | Ensure test attributes can only be set on methods marked with TestMethod attribute |
+| **Category**                        | Usage                                                                              |
+| **Fix is breaking or non-breaking** | Non-breaking                                                                       |
+| **Enabled by default**              | Yes                                                                                |
+| **Default severity**                | Info                                                                               |
+| **Introduced in version**           | 3.3.0 ??                                                                           |
```

Comment:
@Evangelink is this the correct version of testfx that will contain the analyzer ?

Reply:
Confirmed version is 3.3.0.